sim/src/make/generator.rs

```diff
@@ -428,6 +428,56 @@ impl ScenarioGenerator {
         let mut s = Scenario::empty(map, "random people going to/from work");
         s.only_seed_buses = None;
         timer.start_iter("create people", total_ppl);
+
+        let incoming_connections = map.all_incoming_borders();
+        let outgoing_connections = map.all_outgoing_borders();
+        // transit
+        for (_home, _num_ppl) in &residences {
```

Comment:
Scaling the number of these trips based on the number of houses is reasonable I guess. Bit more clear to do something like `for _ in 0..residences.len()` or `for _ in &residences`, the `_` means ignore the value

Reply:
> Scaling the number of these trips based on the number of houses is reasonable I guess.

I have some plans to make it more configurable (per map, some places have heavy transit traffic, some nearly none) - but as the first step I want to have any such traffic and tweak it later.